chat_box/pinecone_db.py

```
@@ -53,26 +64,52 @@ def get_embedding(text: str):
 
 def extract_filter(user_query):
     filter = {}
-    if "ít calo" in user_query.lower() or "giảm cân" in user_query.lower():
-        filter["tags"] = {"$in": ["giảm cân"]}
-    if "tăng cân" in user_query.lower():
-        filter["calories"] = {"$gt": 999}
-    if "nhiều protein" in user_query.lower():
-        filter["protein"] = {"$gt": 999}
+    tags = []
```

Ruff (0.13.3)

Local variable `tags` is assigned to but never used

Remove assignment to unused variable `tags`

(F841)